features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```diff
@@ -480,5 +480,7 @@
   "pagination.max_items_per_page": "100",
   "pagination.default_items_per_page": "15",
 
-  "user_filtering.show_display_name_of_user": false
+  "user_filtering.show_display_name_of_user": false,
+
+  "account_suspension.last_login_as_identity_claim": true
```

Comment:
shall we make this 'identity_mgt.account_suspension.last_login_as_identity_claim' and move it to line 280?